homeassistant/components/zwave_js/api.py

```diff
@@ -1943,14 +1947,16 @@ def forward_progress(event: dict) -> None:
 
     @callback
     def forward_finished(event: dict) -> None:
-        finished: FirmwareUpdateFinished = event["firmware_update_finished"]
+        finished: FirmwareUpdateResult = event["firmware_update_finished"]
         connection.send_message(
             websocket_api.event_message(
                 msg[ID],
                 {
                     "event": event["event"],
                     "status": finished.status,
+                    "success": finished.success,
```

Comment:
Do we need to update the frontend too?

Reply:
we should but it won't break anything in the frontend

Reply:
https://github.com/home-assistant/frontend/pull/13982